README.md

```diff
@@ -73,20 +101,68 @@ user=yourusername
 password=pw
 ```
 
+## Automating the Scraper Runs
+
+There are two ways that the scraper can be automated, via cron, or via Serverless
+with AWS Lambda. No matter how you choose to run it the scraper runs with 
+a `python ttc_scraper_api.py` command.
+
+### AWS Lambda Scraping
+There is a mode which will allow scraping via AWS Lambda 
+with logging added to AWS Cloudwatch. This mode uses the Serverless framework.
+
+The [Serverless](https://serverless.com/) framework is a suite of tooling which allows the 
+easy deployment and management of serverless code.
+
+This allows us to run this code without having to spin up/monitor for an instance 
+manually. And since we only pay for the code when it is running the compute 
+costs are nearly zero.
+
+#### Setup
+In addition to installing the Python requirements (above) 
+we need to install the Serverless framework with npm by running `npm install`
+in the project root. 
+
+Move serverless.yml.template to serverless.yml and replace the bucket name with the actual values
+
+At the time of writing the schedule line in serverless.yml is set as
+
+```yaml
+    rate: cron(* 0-2,5-23 * * ? *)
+```
+which means that it should run every minute from 5am to 2am every day. More 
+information on this cron line can be found on the [AWS documentation](https://docs.aws.amazon.com/lambda/latest/dg/tutorial-scheduled-events-schedule-expressions.html),
+in this documentation references to UTC should be ignored, we use the 
+'serverless-local-schedule' package which allows us to specify crons in local 
+time rather than UTC (otherwise the behaviour would change during daylight 
+savings time).
+
+Tell Serverless which AWS creds you would like to use with  
```

Comment:
I would add how to get these credentials here. Serverless seems to have a handy guide https://serverless.com/framework/docs/providers/aws/guide/credentials/